### ISSUE TITLE
Standardize tooltips placement

### ISSUE THREAD

**Issue by TildaDares** [publiclab/plots2#9836]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file `app/views/like/_like.html.erb` in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -44,7 +44,7 @@
         <% end %>
     <% end %>
 
-    <span data-toggle="tooltip" data-placement="bottom" title="Tools" id="menu-btn" class="btn btn-outline-secondary btn-circle btn-sm" rel="popover" data-placement="left" data-html="true" style="overflow: auto; max-height: 500px" data-content="
+    <span data-toggle="tooltip" data-placement="top" title="Tools" id="menu-btn" class="btn btn-outline-secondary btn-circle btn-sm" rel="popover" data-placement="left" data-html="true" style="overflow: auto; max-height: 500px" data-content="
         <p><b>Users who liked this</b></p>
         <% node.likers.each do |user| %>
             <i class='fa fa-star-o'></i> <a href='/profile/<%= user.username %>/'><%= user.username %></a><br />
```

## Before

https://user-images.githubusercontent.com/63427719/122912680-80ff6e00-d350-11eb-89a7-09e0f9de738d.mov

## After

https://user-images.githubusercontent.com/63427719/122912535-56151a00-d350-11eb-973c-929dac41d93c.mov

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

**Comment by TildaDares** [comment]
@jayita10 Would you like to work on this?

**Comment by Jayita10** [comment]
yes I will work on it

**Comment by TildaDares** [comment]
@Jayita10 There's a fork button at the top of this page.

**Comment by Jayita10** [comment]
Hi, please review my work. Thank you! :)

**Comment by TildaDares** [comment]
@Jayita10 I can't find your pull request.

**Comment by Jayita10** [comment]
"Jayita10 Merge pull request #1 from Jayita10/Jayita10-patch-1
…
d8521e5
12 minutes ago
Merge pull request #1 from Jayita10/Jayita10-patch-1"

this is showing in my pull request tab. 
Please let me know what I did wrong  and what I need to to fix this

**Comment by TildaDares** [comment]
If you're editing the file directly from github then you shouldn't make the changes on your own fork. Find the file on this repo and edit it there. @Jayita10

**Comment by Jayita10** [comment]
Please check now

**Comment by gettinDrikkieWithIt** [comment]
Grabbing this one 😄

**Comment by cesswairimu** [comment]
fixed in #9861
